buildconfig/Jenkins/Conda/conda-cppcheck

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -ex
+
+# This script will set up a conda environment before running the cppcheck shell script.
+#
+# Script usage:
+# conda-cppcheck <job-name> <path-to-workspace>
+#
+# Example command to run a PR build on ubuntu:
+# conda-cppcheck pull_requests-cppcheck /jenkins/workspace_dir/
+#
+# Expected args:
+#   1. JOB_NAME: The name of the job to be run on must include 'pull_requests'.
```

Comment:
Is this comment quite right? If the job name contains "pull_requests" then it does a check if any C++ files have changed but if not it just does the full thing?